test/suite/wttest.py

```diff
@@ -662,6 +662,51 @@ def assertTimestampsEqual(self, ts1, ts2):
     def timestamp_str(self, t):
         return '%x' % t
 
+    def dropUntilSuccess(self, session, uri, config=None):
+        while True:
+            session.checkpoint()
```

Comment:
We are not checking for the return type EBUSY in these functions? If the failure occurred due to some other problem it loops forever. Also, it is better to change the function name to specify EBUSY handling. How about the first trying to perform the operation and then calling the checkpoint if it fails?